src/hb-ot-layout.hh

```diff
@@ -387,7 +387,7 @@ static inline void
 _hb_ot_layout_reverse_graphemes (hb_buffer_t *buffer)
 {
   buffer->reverse_groups (_hb_grapheme_group_func,
-			  buffer->cluster_level == HB_BUFFER_CLUSTER_LEVEL_MONOTONE_CHARACTERS);
+			  HB_BUFFER_CLUSTER_LEVEL_IS_MONOTONE (buffer->cluster_level));
```

Comment:
I think the old cold was wrong in that it wasn't accepting `MONOTONE_GRAPHEMES` here. Let me read the logic some more. I think it does it this way because `MONOTONE_GRAPHEMES` was already applied and taken care of by `_hb_grapheme_grou_func`. Yeah, I think that's right. I'll revert it back.